student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Stundent < Person
+  def initialize(age, name, class_room, parent_permission: true)
```

Comment:
- [ ] The name parameter should have the value `Unknown` assigned to it as default.